benchmarks/benchmark-ab.py

```diff
@@ -270,6 +270,7 @@ def generate_csv_output():
 
     with open('/tmp/benchmark/predict.txt') as f:
         lines = f.readlines()
+        lines.sort(key=float)
```

Comment:
Since you already have Pandas imported here, it would be simpler to use the [quantile](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.quantile.html) function instead of all this calculation.